x/emissions/proto/emissions/v5/params.proto

```diff
@@ -152,4 +152,7 @@ message Params {
     (gogoproto.customtype) = "github.com/allora-network/allora-chain/math.Dec",
     (gogoproto.nullable) = false
   ];
+  bool global_whitelist_enabled = 50; // global whitelist enabled => all global whitelisted actors can create topics
+  // and participate in all topics as workers and reputers
+  bool topic_creator_whitelist_enabled = 51; // topic creator whitelist enabled => only topic creator whitelisted actors can create topics
```

Comment:
We could consider adding something to globally turn on all whitelists, or an emergency switch to shut off all activity entirely. @spooktheducks @relyt29 thoughts? imo not the biggest fan of this as it could make a chilling effect but could be fine for a young network.